models/api.model.js

```diff
@@ -0,0 +1,16 @@
+const db = require('../db/connection')
+const endpoints = require('../endpoints.json')
+
+exports.findApi = () => {
+
+    //make sure each endpoint in the file has a queries and a exampleResponse key
+    for(const key in endpoints){
+        if(!endpoints[key].hasOwnProperty('queries')){
+            endpoints[key].queries = []
+        }
+        if(!endpoints[key].hasOwnProperty('exampleResponse')){
+            endpoints[key].exampleResponse = {}
+        }
+    }
```

Comment:
Some of this logic isn't necessary, you can count on const / require to pull in the contents of the _endpoints_ file so there shouldn't be any need to validate the contents before sending it back